assets/src/js/Components/ArchivePicker/ArchivePicker.js

```diff
@@ -398,6 +399,30 @@ export default function ArchivePicker({view = ADMIN_VIEW}) {
     }
   }, [currentBlock]);
 
+  useEffect(() => {
+    const sideBarElements = document.getElementsByClassName('components-panel__body');
+
+    for (let i = 0; i < sideBarElements.length; i++) {
+      const featuredImageDiv = sideBarElements[i].querySelector('.editor-post-featured-image');
+      if (featuredImageDiv) {
+        const featuredImageDivCtn = featuredImageDiv.querySelector('.editor-post-featured-image__container');
```

Comment:
Why don't directly do something like this?

```suggestion
        const featuredImageDivCtn = featuredImageDiv.querySelector('.editor-post-featured-image__container .editor-post-featured-image__toggle');
```